release-notes/aws/v9.0.6.md

```diff
@@ -20,3 +20,9 @@ Below, you can find more details on components that were changed with this relea
 
 ### Kubernetes [v1.15.12](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.15.md#changelog-since-v11511)
 - Updated from v1.15.11.
+
+### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.12](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v1612-2020-06-04))
+
+- Made healthcheck probes configurable.
+- Made liveness probe more resilient.
+- Align labels, use `app.kubernetes.io/name` instead of `k8s-app` where possible. `k8s-app` remains to be used for compatibility reasons, as selectors are not modifiable without recreating the Deployment.
```

Comment:
```suggestion
- Aligned labels using `app.kubernetes.io/name` instead of `k8s-app` where possible. `k8s-app` remains to be used for compatibility reasons as selectors are not modifiable without recreating the Deployment.
```